Purchases/Purchasing/StoreKitAbstractions/SubscriptionPeriod.swift

```diff
@@ -66,6 +66,20 @@ public class SubscriptionPeriod: NSObject {
 
 }
 
+// MARK: - Renames
+
+// @available annotations to help users migrating from `SKProductSubscriptionPeriod` to `SubscriptionPeriod`
+public extension SubscriptionPeriod {
+
+    /// The number of units per subscription period
+    @available(iOS, obsoleted: 1, renamed: "value")
```

Comment:
I'd change this to unavailable. I think it's essentially the same but semantically more correct since it was never available?

Reply:
oops, I'd already merged, I'll make a new PR